src/core/Akka.Streams.Tests/Dsl/HubSpec.cs

```diff
@@ -276,6 +277,36 @@ await EventFilter.Error(contains: "Upstream producer failed with exception").Exp
             }, Materializer);
         }
 
+         [Fact]
+        public async Task MergeHub_must_not_log_normal_shutdown_exception()
+        {
+            await this.AssertAllStagesStoppedAsync(async () =>
+            {
+                var (sink, task) = MergeHub.Source<int>(16).Take(10).ToMaterialized(Sink.Seq<int>(), Keep.Both).Run(Materializer);
+
+                await WithinAsync(10.Seconds(), async () =>
+                {
+                    await EventFilter
+                        .Custom((e) => 
+                        { 
+                            if (e.Cause?.InnerException is NormalShutdownException nse && nse == ActorPublisher.NormalShutdownReason)
+                                return true;
+                            else 
+                                return false;
+                        })
+                        
+                    // await EventFilter.Error(contains: ActorPublisher.NormalShutdownReasonMessage)
+                        .ExpectAsync(0, async () =>
```

Comment:
LGTM

Reply:
I need to cleanup those comments though

Reply:
Good idea - I mostly mean that I grokked your assertion and it looks correct for what you're trying to achieve